samples/Samples.AspNetCoreMvc31/Properties/launchSettings.json

```diff
@@ -8,46 +8,81 @@
     }
   },
   "profiles": {
-    "IIS Express": {
+    "StartupHook IIS Express": {
       "commandName": "IISExpress",
-      "launchBrowser": false,
+      "launchBrowser": true,
```

Comment:
It is just a convenience since it shows a landing page with some useful information and you don't need to know which port it is running 😬 that said I don't have a strong preference, either way works for me.